code/_onclick/hud/blob_overmind.dm

```diff
@@ -53,8 +53,13 @@
 
 /atom/movable/screen/blob/blobbernaut
 	icon_state = "ui_blobbernaut"
-	name = "Produce Blobbernaut (40)"
-	desc = "Produces a strong, smart blobbernaut from a factory blob for 40 resources.<br>The factory blob used will become fragile and unable to produce spores."
+	name = "Produce Blobbernaut (ERROR)"
+	desc = "Produces a strong, smart blobbernaut from a factory blob for (ERROR) resources.<br>The factory blob used will become fragile and unable to produce spores."
+
+/atom/movable/screen/blob/blobbernaut/Initialize()
+	..()
+	name = "Produce Blobbernaut ([BLOBMOB_BLOBBERNAUT_RESOURCE_COST])"
+	desc = "Produces a strong, smart blobbernaut from a factory blob for [BLOBMOB_BLOBBERNAUT_RESOURCE_COST] resources.<br>The factory blob used will become fragile and unable to produce spores."
```

Comment:
If you are worried about one being updated and not the other, instead of handling this in initialization you can make new defines near each other:
```dm
#define BLOB_STRUCTURE_RESOURCE_COST 40
#define BLOBBERNAUT_NAME "Produce Blobbernaut (40)"
#define BLOBBERNAUT_DESC ""Produces a strong, smart blobbernaut from a factory blob for 40 resources.\nThe factory blob used will become fragile and unable to produce spores.""
```

Reply:
I don't see the benefit to doing this, as compared to changing it on initialize with the accurate values?